src/components/monitor/monitor.jsx

```diff
@@ -29,7 +29,10 @@ const modes = {
 };
 
 const MonitorComponent = props => (
-    <ContextMenuTrigger id={`monitor-${props.label}`}>
+    <ContextMenuTrigger
+        holdToDisplay={props.mode === 'slider' ? -1 : 1000}
```

Comment:
Just so I understand — this is the prop that lets long-press open the menu? So 1s hold to open?

Reply:
Yup, which is the default. The more important part is that it is disabled for sliders.